calcom/cal.com
# fix: Embed Snippet Generator, hard crash on reload

## What does this PR do?

![image](https://github.com/calcom/cal.com/assets/1780212/8a9e26c8-5af2-4000-b6c8-8adced2065fa)

[After fix Loom](https://www.loom.com/share/21b06b8ab4b6435991dd027898966ebe)

## Type of change

## How should this be tested?
- Go to an event-type and open inline embed code and refresh to see the crash. See the loom shared above.
## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works


<sub>[CAL-2469](https://linear.app/calcom/issue/CAL-2469/fix-embed-snippet-generator-hard-crash-on-reload)</sub>
